sources/packages/web/src/views/student/financial-aid-application/Applications.vue

```diff
@@ -25,6 +28,15 @@ export default {
   setup() {
     const router = useRouter();
     const toast = useToastMessage();
+    const sinValidStatus = ref(1);
+    const store = useStore();
+
+    onMounted(async () => {
+      sinValidStatus.value = computed(
```

Comment:
The computed property does not need another ref variable. Can you please check other places where the computed was created and used?